week-3/B-array-some/exercise.js

```diff
@@ -16,6 +16,9 @@ var students = ["Islam", "Lesley", "Harun", "Rukmini"];
 var mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
 var pairs = pairsByIndex.map(function(indexes) {
+  if (indexes === null) {
+    process.exit(1);
```

Comment:
Well, that's clever! Indeed this is doing what the exercise was asking :) However, I'd like you to not alter this code at all, and write your function on line 10. Please use the array method `some()` as well, thanks!